src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -190,17 +192,15 @@ func parseFlags() {
 	}
 	// Note - we can ignore the returned error because CommandLine.Parse() will
 	// exit if it fails.
-	flag.CommandLine.Parse(args)
+	_ = flag.CommandLine.Parse(args)
```

Comment:
No if we want errcheck to pass. We should explicitly discard errors from now on.